src/requests/project.js

```diff
@@ -0,0 +1,16 @@
+const config = require('../config')
+const http = require('../support/http')
+
+const getProject = (pathParams, queryParams, accept = 'json') => {
+    const { SWAGGERHUB_URL, SWAGGERHUB_API_KEY } = config.getConfig()
```

Comment:
Even though it's not a "spec", would it work to reuse `getSpec()` here?
If that does work, perhaps we should rename spec.js to something more generic like request.js

Reply:
Can confirm that getSpec works as a drop in replacement here so it's definitely worth looking into. This would just mean refactoring the existing wrappers under the requests package i.e. src/requests/api.js to use the new request.js (and presumably updated functions such as getRequest etc)